.github/workflows/autobot.yml

```diff
@@ -0,0 +1,33 @@
+name: automatically-update-test-links
```

Comment:
I would name the file to something more specific, like `workflows/update-test-images.yml`